.circleci/config.yml

```diff
@@ -60,7 +60,8 @@ jobs:
       - image: cimg/openjdk:17.0.0
     steps:
       - run: git clone https://github.com/openmf/ph-ee-integration-test
-      - run: cd ph-ee-integration-test && ./gradlew test -Dcucumber.filter.tags="@gov" 
+      - run: cd ph-ee-integration-test && ./gradlew test -Dcucumber.filter.tags="@gov"
+      - run: ./gradlew test -Dcucumber.filter.tags="@amsintegrationtest"
```

Comment:
Instead of adding this as an additional command, add this as an additional step. it will be easier to track.